### DISCUSSION TITLE
The term 'email-auth-node' is not recognized as the name of a cmdlet, function

### DISCUSSION THREAD

**Discussion by hariprasath17699** [saran-surya/email_auth_node#15] [Q&A]
email-auth-node : The term 'email-auth-node' is not recognized as the name of a cmdlet, function, 
script file, or operable program. Check the spelling of the name, or if a path was included, verify 
that the path is correct and try again.
At line:1 char:1
+ email-auth-node --version
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (email-auth-node:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Im getting this error can anyone solve this?

**Comment by saran-surya** [comment]
Hi @hariprasath17699 , so have you performed the
```npm link``` Operation ?

**Comment by hariprasath17699** [comment]
Yes that issue solved but email is not sending

**Reply by saran-surya** [reply]
Please send in the logs. Or what error you are correctly facing.
